run_local.js

```diff
@@ -25,6 +23,13 @@ async function main() {
                 status: 200,
                 headers: {},
             }),
+    };
+
+    runServer({
+        name: "DEV_LOCAL",
+        port: 8080,
+
+        renderEnvironment,
```

Comment:
This hopefully illustrates a little how this shifts the architecture. Instead of passing a single function for rendering and having the gateway manage everything around that, we now pass an environment that is responsible for that management, providing us a neat abstraction.